packages/ui-library/src/components/select/index.css.ts

```diff
@@ -1,63 +1,147 @@
-import { typeSafeNestedCss } from "../../utils/nested-typesafe-css-literals";
+import { typeSafeNestedCss } from "../../utils/css-in-ts/nested-typesafe-css-literals";
 
-import { renderThemedCssStrings } from "../../foundation/_tokens-generated/index.pseudo.generated";
+import { SemanticThemeIterator, ComponentThemeIterator } from "../../foundation/_tokens-generated/index.pseudo.generated";
 
-export const styleCustom = typeSafeNestedCss`
+export const staticStyles = typeSafeNestedCss/*css*/ `
   .blr-select-option {
     display: flex;
     flex-direction: column;
     align-items: flex-start;
   }
-`;
 
-export const { tokenizedLight: selectInputLight, tokenizedDark: selectInputDark } = renderThemedCssStrings(
-  (_componentTokens, semanticTokens) => {
-    const { inputfield, inputslot, labelslot } = semanticTokens.sem.forms;
-    const { InputIcon } = _componentTokens.cmp;
+  .blr-select > .label-wrapper {
+    display: flex;
+  }
 
-    return typeSafeNestedCss`
-      :host {
-        .blr-select {
-          &.sm {
-            & > .label-wrapper {
-              display: flex;
-              padding: ${labelslot.padding.sm};
-            }
-          }
-          &.md {
-            & > .label-wrapper {
-              display: flex;
-              padding: ${labelslot.padding.md};
-            }
-          }
-          &.lg {
-            & > .label-wrapper {
-              display: flex;
-              padding: ${labelslot.padding.lg};
-            }
-          }
+  slot {
```

Comment:
why is slot hidden? If there is no use for it, then we better remove the slot completely.

Reply:
out of scope for this pr